README.md

```diff
@@ -7,11 +7,6 @@ Live app: https://sophiali.dev/eat-play-drink/
 - User can filter between places to eat, play, drink
 - User can reset the map locations
 
-## Future Application Features
```

Comment:
Nice! So as a tip for repo work, in the event you want to backlog these ideas, the best way is to simply create Issues for each one and label it with something like "backlog". This then allows you to communicate that you've thought these things through, but currently will not prioritize them for development.

Another benefit you get from this is that in case anyone else wants to help build it for you / add feedback, it's easy for them to do so!

Reply:
Putting ideas in a backlog definitely makes the README cleaner. I've removed the ideas from the README and into issues marked with "backlog". Thanks for the pro-tip!